test/functional/test-ads-config.js

```diff
@@ -18,6 +18,17 @@ import {adConfig} from '../../ads/_config';
 
 describe('test-ads-config', () => {
 
+  it('should have all ad networks configured', () => {
+    window.ampTestRuntimeConfig.adTypes.forEach(adType => {
```

Comment:
We tested that all ad networks have been configured. Do we also need to test if all configurations are about one supported ad network?